### ISSUE TITLE
Feature Request: File level Deduplication for storage optimization 

### ISSUE THREAD

**Issue by draft1** [andreimarcu/linx-server#209]
* File level [Deduplication](http://en.wikipedia.org/wiki/Data_deduplication) for storage optimization (does store duplicate files only once, but generate multiple links)
* Idea taken from this script: https://gitlab.com/mojo42/Jirafeau

Thanks!

**Comment by andreimarcu** [comment]
Out of scope for this project, I suggest looking into deduplicating filesystems

**Comment by draft1** [comment]
Okay, but still very good to have feature.
I hope to have this in the future version, thanks Sir.
